client/verta/verta/_uac/_role.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+from verta._protos.public.uac import RoleV2_pb2 as _Role
```

Comment:
See my comment in `_organization.py`

```suggestion
from verta._protos.public.uac import RoleV2_pb2
```